src/qt/forms/overviewpage.ui

```diff
@@ -546,21 +661,87 @@
                          </property>
                         </widget>
                        </item>
-                       <item row="2" column="0">
-                        <widget class="QLabel" name="labelPendingText">
+                       <item row="4" column="0">
+                        <widget class="QLabel" name="labelLockedBalanceText">
+                         <property name="toolTip">
+                          <string>Locked PIV or Masternode collaterals. These are excluded from zPIV minting.</string>
+                         </property>
                          <property name="text">
-                          <string>Pending:</string>
+                          <string>Locked:</string>
                          </property>
                         </widget>
                        </item>
-                       <item row="0" column="1">
-                        <widget class="QLabel" name="labelSpendable">
+                       <item row="4" column="1">
+                        <widget class="QLabel" name="labelLockedBalance">
+                         <property name="toolTip">
+                          <string>Locked PIV or Masternode collaterals. These are excluded from zPIV minting.</string>
+                         </property>
                          <property name="text">
-                          <string>Spendable:</string>
+                          <string>0.000 000 00 PIV</string>
```

Comment:
should be `<string notr="true">0.000 000 00 PIV</string>` to prevent sending this to transifex for translation

Reply:
Same here.